app/models/issuer.py

```diff
@@ -98,3 +102,39 @@ class ClearPendingRevocationsResult(BaseModel):
             "The resulting revocations that are still pending after a clear-pending request has been completed."
         ),
     )
+
+
+class RevokedResponse(BaseModel):
+    cred_rev_ids_published: Dict[str, List[int]] = Field(
+        default_factory=dict,
+        description=(
+            "A map of revocation registry IDs to lists of credential revocation IDs "
+            "(as integers) that have been revoked. Can be empty."
+        ),
+    )
+
+    @model_validator(mode="before")
+    @classmethod
+    def extract_revoked_info(
+        cls, values: TxnOrPublishRevocationsResult
+    ) -> Dict[str, Any]:
+        if isinstance(values, dict) and "txn" in values:
+            txn_list: List[TransactionRecord] = values.get("txn")
```

Comment:
minor note: this isn't actually a List[TransactionRecord]. It's a List[Dict[str,Any]], which has same shape as List[TransactionRecord]. Point being that the pydantic model doesn't have a .get method. So the type hint makes it seem like `txn.get` will error. So, for clarity we can just set the type hint to list of dict, and comment that it's list transactionrecord